Description: Disable git versioning.
 This disables a git function that sets version as we are working
 with tarball releases and will not have a git repo.
Author: Scarlett Moore <sgmoore@kde.org>
Forwarded:  not-needed
Last-Update: 2019-04-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4b8aeec..7038665 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,6 @@ cmake_minimum_required(VERSION 3.2)
 project(AppImageLauncher)

 # versioning
-include(cmake/versioning.cmake)
+#include(cmake/versioning.cmake)
